projects/hwloc/index.php

```diff
@@ -142,9 +142,11 @@
 or <a href="git.php">checkout</a>.
 </p>
 
-<p> The <b>version string of the latest release or release-candidate is available
- from <a href="../../software/hwloc/current/downloads/latest_snapshot.txt">here</a></b>.
- For the latest on a specific series, replace "current" with "v2.4" for instance.
+<p> The <b>version string of the latest release is available from the
+ <a href="../../software/hwloc/current/downloads/latest_release.txt"><tt>latest_release.txt</tt></a> link</b>.
+ For the latest release or release candidate, rather use the
+ <a href="../../software/hwloc/current/downloads/latest_snapshot.txt"><tt>latest_snapshot.txt</tt></a> link.
+ For the latest on a specific series, replace "<tt>current</tt>" with "<tt>v2.9</tt>" in these links for instance.
```

Comment:
Do you really want to hard-code a specific version here?  (I know you're just carrying forward older text -- should we make this variadic somehow?)

Reply:
You mean replace "v2.9" with something like "<?php print($series_dir); ?>" ?
In general, I don't care much. However, once 3.0 gets released, I may actually prefer hardwiring "v2.10" for people that cannot upgrade yet.

Reply:
If you're cool with it (the hard-coded `v2.9`), then so am I.  I just always ask when I see hard-coded version numbers, because they typically don't get updated until someone happens to notice that they're out of date (like now 😄).

Reply:
Oh, hahah!  Your PHP got rendered as "", not:
```php
<?php print($series_dir); ?>
```

Either way is fine with me.  If you want to update the PR, cool.  If you don't care, fine.